platform_versioned_docs/version-23.4.0/data/data-studios.mdx

```diff
@@ -41,7 +41,7 @@ This functionality is available to users with the **Maintain** role and above.
 You'll be returned to the Data Studios landing page that displays the list of data studios in your workspace. The data studio you created will be listed with the status **stopped**.
 
 :::note
-By default, Data Studios has read and write permissions to any mounted data path. This means that data can be written from the data studio session back to the cloud storage path mounted. To stop potential data loss, only one data studio session per workspace can mount a unique data path. When adding a new data studio, data paths already mounted to other running data studios are unavailable. If a new file is uploaded to the cloud storage bucket path while a session is running, the file may not be available to the data studio session immediately.
+By default, Data Studios only has read permissons to mounted data paths. Write permissions can be added for specific buckets during the AWS Batch compute environment set up, by specifying additional **Allowed S3 Buckets**. This means that data can be written from the data studio session back to the cloud storage path mounted. To stop potential data loss, only one data studio session per workspace can mount a unique data path. When adding a new data studio, data paths already mounted to other running data studios are unavailable. If a new file is uploaded to the cloud storage bucket path while a session is running, the file may not be available to the data studio session immediately.
```

Comment:
Remove AWS Batch specification (limit number of places that we specify cloud providers to just the initial admonishment)

```suggestion
By default, Data Studios only has read permissions to mounted data paths. Write permissions can be added for specific cloud storage buckets during the compute environment configuration by defining additional **Allowed S3 Buckets**. This means that data can be written from the data studio session back to the cloud storage path mounted. To stop potential data loss, only one data studio session per workspace can mount a unique data path. When adding a new data studio, data paths already mounted to other running data studios are unavailable. If a new file is uploaded to the cloud storage bucket path while a session is running, the file may not be available to the data studio session immediately.
```